src/components/CategoriesList.js

```diff
@@ -14,7 +14,7 @@ export const CategoriesList = (props) => {
       {
         categories
           ? categories.map(category => <Category key={category.name} name={category.name} id={category.id} />)
-          : null 
+          : props.categoriesError ? 'Sorry, there seems to be an error. Please try again later.' : null
       }
```

Comment:
should we have a if statement instead of a ternary inside of a ternary operator?